packages/platform/types/users/users_2024_06_18/inputs/update-user.input.ts

```diff
@@ -0,0 +1,142 @@
+import {
+  IsEmail,
+  IsOptional,
+  IsString,
+  registerDecorator,
+  IsInt,
+  ValidatorConstraint,
+  IsBoolean,
+} from "class-validator";
+import type { ValidationOptions, ValidatorConstraintInterface } from "class-validator";
+
+import { checkUsername } from "@calcom/lib/server/checkUsername";
```

Comment:
this should be imported/exported within platform/libraries package, and then imported from there, remember to run yarn build inside platform/libraries when making change there